### PR TITLE
ci: add a GitHub workflow to run citgm-all

### PR DESCRIPTION
**Pull request by targos**

### CODE REVIEW

File: .github/workflows/test-all.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      - name: Use Node.js ${{ github.event.inputs.node_version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ github.event.inputs.node_version }}
+
+      - name: Install packages
+        run: npm install
+
+      - name: Run citgm-all
+        run: node bin/citgm-all.js ${{ github.event.inputs.args }}
```
Comment (targos) [inline .github/workflows/test-all.yml:60]: This doesn't work if there are multiple space-separated arguments, because they end up as only one element in `process.argv`. Any idea how we could fix that?

Comment (targos) [reply .github/workflows/test-all.yml:60]: Wait, actually I just realized that I didn't use the right input name... Let me try again

**Comment by targos** [comment]
Test with `--includeTags lodash`: https://github.com/targos/citgm/actions/runs/629355250

File: .github/workflows/test-all.yml (hunk 1 above)
Comment (targos) [reply .github/workflows/test-all.yml]: Okay, it works!

**Comment by targos** [comment]
@nodejs/citgm PTAL

**Comment by MylesBorins** [comment]
Hey all, I made a boo boo on main and had to force push. I've rebased this branch and force pushed to make sure that you don't have to do extra work because of my mistake
